commands/publish/index.js

```diff
@@ -144,10 +144,58 @@ class PublishCommand extends Command {
         )
       : [this.packagesToPublish];
 
+    const isBreakingChange = (currentVersion, nextVersion) => {
```

Comment:
This should be a function external to the class, there's no reason to inline it here.

Reply:
we discussed it in last code review, I basically can't find a place to host it so I inlined it. Any suggestions?

Reply:
```function isBreakingChange(currentVersion, nextVersion) {
  ...